spring-jdbc/documentation.md

```diff
@@ -0,0 +1,166 @@
+# The Crate JDBC Sample Application
+
+**Note:** The application is written in Java 11.
+
+## Installation
+
+In this sample application, we use the [Crate JDBC driver 2.1.7][3] which
+uses [PostgreSQL wire protocol v3][1]. To obtain the [Crate JDBC driver][4],
+please follow [instructions][5] for the build tool of your choice.
+
+## Usage
+
+### Connecting to Crate
+
+The application establishes the connection with Crate using a Spring JDBC datasource configured in the `application.properties` file:
+
+```text
+spring.datasource.url=jdbc:crate://localhost:5432/
+spring.datasource.username=crate
+spring.datasource.password=
+spring.datasource.driver-class-name=io.crate.client.jdbc.CrateDriver
+```
+
+Please, take a look at the Crate JDBC driver documentation to see the
+[possible forms][6] of URLs supported by the driver.
+
+### Executing statements
+
+For all statements Spring's JDBC template is used. E.g reading a blog post from CrateDB:
+
+```java
```

Comment:
What is the purpose of copying over the code snapshots to the README? It will most definitely be unsynced pretty soon. Can we maybe refer to some interfaces or just stick to SQL only?